src/modules/telemetry/implementation/quest-telemetry.js

```diff
@@ -0,0 +1,45 @@
+import { Sender } from '@questdb/nodejs-client';
+
+class QuestTelemetry {
+    async initialize(config, logger) {
+        this.config = config;
+        this.logger = logger;
+    }
+
+    listenOnEvents(eventEmitter, onEventReceived) {
+        return eventEmitter.on('operation_status_changed', onEventReceived);
+    }
+
+    async sendTelemetryData(
+        operationId,
+        timestamp,
+        blockchainId = '',
+        name = '',
+        value1 = null,
+        value2 = null,
+        value3 = null,
+    ) {
+        try {
+            const sender = Sender.fromConfig(this.config.ip_endpoint);
```

Comment:
Maybe move this to be filed in class so it doesn't need to be initialized every time function is called